pkg/controller/v1alpha1/inferencegraph/raw_ig.go

```diff
@@ -90,6 +90,9 @@ func createInferenceGraphPodSpec(graph *v1alpha1api.InferenceGraph, config *Rout
 		}
 	}
 
+	if graph.GetAnnotations()[constants.ODHKserveRawAuth] == "true" {
```

Comment:
To me, it is too strict to set `true` I mean user can set True as well.

So why don't we use  `strconv.ParseBool` for the annotation?

Reply:
removed this section